geode-core/src/main/java/org/apache/geode/internal/cache/entries/DiskEntry.java

```diff
@@ -1623,6 +1623,10 @@ public static void updateVersionOnly(DiskEntry entry, InternalRegion region, Ver
       }
     }
 
+    static boolean doSynchronousWrite(InternalRegion region, DiskRegion dr) {
+      return dr.isSync() || (dr.isBackup() && !region.isInitialized());
```

Comment:
Either rename DiskRegion.isBackup() or file a ticket to rename that method. Currently, it's easy to confuse this with Backups in some way.